server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/service/UserQuotaService.java

```diff
@@ -37,10 +40,12 @@ public UserQuotaService(MaxQuotaManager maxQuotaManager) {
         this.maxQuotaManager = maxQuotaManager;
     }
 
-    public void defineQuota(String user, QuotaDTO quota) throws MailboxException {
+    public void defineQuota(String user, QuotaDTO quota) {
         QuotaRoot quotaRoot = QuotaRoot.forUser(user);
```

Comment:
As explained in another review, this method should not exist.

You should rely on the **QuotaRootResolver**.

As QuotaRoot is a mailbox level property, not necessarilly corelated to user, we should define a `user quota root` as `the quota root of a user INBOX` IMO. CF: https://github.com/linagora/james-project/pull/1308 for full justification.

Reply:
fixed